hcn/hcnglobals.go

```diff
@@ -50,7 +50,13 @@ var (
 		VersionRange{MinVersion: Version{Major: 10, Minor: 4}, MaxVersion: Version{Major: math.MaxInt32, Minor: math.MaxInt32}},
 	}
 	// HNS 11.10 allows for session affinity for loadbalancing
-	SessionAffinityVersion = VersionRanges{VersionRange{MinVersion: Version{Major: 11, Minor: 10}, MaxVersion: Version{Major: math.MaxInt32, Minor: math.MaxInt32}}}
+	SessionAffinityVersion = VersionRanges{VersionRange{MinVersion: Version{Major: 12, Minor: 0}, MaxVersion: Version{Major: math.MaxInt32, Minor: math.MaxInt32}}}
+
+	// HNS 10.5 through 11 not included and 12.0+ supports Ipv6 dual stack.
+	IPv6DualStackVersion = VersionRanges{
```

Comment:
Does this mean the feature is not supported in 10.5+ other than 11.x?

Reply:
Currently we are backporting this feature only to 19H1. When it is backported to all versions of the OS we can relax this check

Reply:
Let me know if you see any issue with this

Reply:
Okay, so 12.0+ is the version for the release where this was developed, and 10.5-10.x is where it's backported to?

Reply:
The "not included" phrasing in the comment above makes it sound to me like 10.5 does _not_ support this feature. Could the comment be clarified?